packages/flutter/lib/src/cupertino/checkbox.dart

```diff
@@ -159,8 +252,23 @@ class CupertinoCheckbox extends StatefulWidget {
   /// [RoundedRectangleBorder] with a circular corner radius of 4.0.
   final OutlinedBorder? shape;
 
+  /// True if this checkbox wants to show an error state.
+  ///
+  /// The checkbox will have different default container color and check color
+  /// when this is true.
+  ///
+  /// Defaults to false.
+  final bool isError;
+
+  /// The semantic label for the checkbox that will be announced by screen readers.
+  ///
+  /// This is announced in accessibility modes (e.g TalkBack/VoiceOver).
+  ///
+  /// This label does not show in the UI.
+  final String? semanticLabel;
+
   /// The width of a checkbox widget.
-  static const double width = 18.0;
+  static const double width = 14.0;
```

Comment:
Is it only the width or width and height (since it's mostly a square?)

Reply:
Width and height.